MainModule/Server/Commands/Players.lua

```diff
@@ -935,14 +935,14 @@ return function(Vargs, env)
 				local adminDictionary, workspaceInfo = nil, nil
 				if elevated then
 					adminDictionary = {}
-					for _, v in pairs(service.GetPlayers()) do
+					for _, v in ipairs(service.GetPlayers()) do
 						local level, rank = Admin.GetLevel(v)
 						if level > 0 then
 							adminDictionary[v.Name] = rank or "Unknown"
 						end
 					end
 					local nilPlayers = 0
-					for _, v in pairs(service.NetworkServer:GetChildren()) do
+					for _, v in ipairs(service.NetworkServer:GetChildren()) do
```

Comment:
Isn't `ipairs()` less performance than `pairs()`? as `ipairs()` skips nil values?

service.GetPlayers will never return nil values in it's table. Using `ipairs()` has no benefit.